src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactDetailsHeader.tsx

```diff
@@ -1,29 +1,127 @@
-import { Box } from '@material-ui/core';
-import React from 'react';
+import { Avatar, Box, IconButton, styled, Typography } from '@material-ui/core';
+import { Close, MoreVert } from '@material-ui/icons';
+import { Skeleton } from '@material-ui/lab';
+import React, { ReactElement } from 'react';
+import { useTranslation } from 'react-i18next';
+import theme from '../../../../theme';
 
+import { StarContactIcon } from '../../StarContactIcon/StarContactIcon';
 import { useGetContactDetailsHeaderQuery } from './ContactDetailsHeader.generated';
+import { ContactHeaderAddressSection } from './ContactDetailsHeaderSection/ContactHeaderAddressSection';
+import { ContactHeaderPhoneSection } from './ContactDetailsHeaderSection/ContactHeaderPhoneSection';
+import { ContactHeaderEmailSection } from './ContactDetailsHeaderSection/ContactHeaderEmailSection';
+import { ContactHeaderStatusSection } from './ContactDetailsHeaderSection/ContactHeaderStatusSection';
 
 interface Props {
   accountListId: string;
   contactId: string;
 }
 
-export const ContactDetailsHeader: React.FC<Props> = ({
+const HeaderBar = styled(Box)(({}) => ({
+  display: 'flex',
+  paddingBottom: 24,
```

Comment:
There's a lot of magic numbers in here. Would the `theme.spacing(...)` function be useful here?

Reply:
Or grid might be useful for arranging, that has a spacing option